Example/Gemfile

```diff
@@ -1,11 +1,3 @@
-def cocoapods_gem(name, gem_name = name.downcase, **opts)
-  gem gem_name, git: "https://github.com/CocoaPods/#{name}", **opts
-end
-
 source 'https://rubygems.org' do
-  cocoapods_gem 'CocoaPods', branch: 'master'
-  cocoapods_gem 'Core', 'cocoapods-core'
-  cocoapods_gem 'Molinillo'
-  cocoapods_gem 'Nanaimo'
-  cocoapods_gem 'Xcodeproj'
+	gem 'cocoapods', '1.10.0'
```

Comment:
Is it a `tab`? The space looks extra wide, like 6 spaces on GitHub, but seems fine locally 🤔

Reply:
Yeah, it's a tab. Not sure why it shows up like this in the diff. 🤔 I'll change it to use 2 spaces instead, to match the old style.